### PR TITLE
techdocs: make some small improvements to faq

### PR DESCRIPTION
**Pull request by dufcrule**
## Hey, I just made a Pull Request!

Made some small improvements to the FAQ. These include using the correct way to write RST and AsciiDoc, a link to the MKDocs Material Theme repo and some small style/grammar changes to the text about file formats.

#### :heavy_check_mark: Checklist

- [ ] All tests are passing `yarn test`
- [ ] Screenshots attached (for UI changes)
- [x] Relevant documentation updated
- [ ] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

File: docs/features/techdocs/FAQ.md [hunk 1]
```diff
@@ -11,14 +11,14 @@ This page answers frequently asked questions about [TechDocs](README.md).
...
-- [Does TechDocs support file formats other than Markdown (e.g. rst, asciidoc)?](#does-techdocs-support-file-formats-other-than-markdown-eg-rst-asciidoc-)
+- [Does TechDocs support file formats other than Markdown (e.g. RST, AsciiDoc)?](#does-techdocs-support-file-formats-other-than-markdown-eg-rst-asciidoc-)
 
 #### What static site generator is TechDocs using?
 
 TechDocs is using [MkDocs](https://www.mkdocs.org/) to build project
 documentation under the hood. Documentation built with the
 [techdocs-container](https://github.com/spotify/backstage/blob/master/packages/techdocs-container/README.md)
-is using the MkDocs Material Theme.
+is using the MkDocs [Material Theme](https://github.com/squidfunk/mkdocs-material).
```
Comment (OrkoHunter) [inline docs/features/techdocs/FAQ.md:21]:
```suggestion
is using the MkDocs
[Material Theme](https://github.com/squidfunk/mkdocs-material).
```

File: docs/features/techdocs/FAQ.md [hunk 2]
```diff
@@ -29,9 +29,9 @@ plugins (e.g.
...
 
-#### Does TechDocs support file formats other than Markdown (e.g. rst, asciidoc) ?
+#### Does TechDocs support file formats other than Markdown (e.g. RST, AsciiDoc) ?
 
 Not right now. We are currently using MkDocs to generate the documentation from
-source. So, they have to be in Markdown format. However, in future we want to
-support other alternatives to MkDocs. That will make it possible to use other
+source, so the files have to be in Markdown format. However, in the future we want to
+support other static site generators which will make it possible to use other
 file formats.
```
Comment (OrkoHunter) [inline docs/features/techdocs/FAQ.md:37]:
```suggestion
source, so the files have to be in Markdown format. However, in the future we
want to support other static site generators which will make it possible to use
otherfile formats.
```

**Review by OrkoHunter — APPROVED** [review]
Thank you!

File: docs/features/techdocs/FAQ.md (hunk 2 above)
Comment (dufcrule) [reply docs/features/techdocs/FAQ.md]: Oops! Just noticed here that "other file" should be two separate words. This is how it's being rendered at https://backstage.io/docs/features/techdocs/faqs:

techdocs

**Pull request merged**